generateAssets.js

```diff
@@ -0,0 +1,126 @@
+const { Octokit } = require('@octokit/rest');
+const fs = require('fs/promises');
+const createActionAuth = require('@octokit/auth-action');
+
+const fullLogoLight = 'https://raw.githubusercontent.com/Draco-lang/Language-suggestions/main/Resources/Logo-Long.svg';
+const fullLogoDark = 'https://raw.githubusercontent.com/Draco-lang/Language-suggestions/main/Resources/Logo-Long-Inverted.svg';
+const shortLogoUrlLight = 'https://raw.githubusercontent.com/Draco-lang/Language-suggestions/main/Resources/Logo-Short.svg';
+const shortLogoUrlDark = 'https://raw.githubusercontent.com/Draco-lang/Language-suggestions/main/Resources/Logo-Short-Inverted.svg';
+const githubLogoDark = 'prebuild_assets/github-mark-white.svg';
+const githubLogoLight = 'prebuild_assets/github-mark.svg';
```

Comment:
Since you have these "pairs" of assets, why not introduce it with some structure? Something like `ThemedResource { lightUrl, darkUrl }`

Reply:
Sir it's JS, not Typescript, there are no interfaces/types.
Will do an object but it's a bit a footgun without type checking.